### PR TITLE
OneNote (Personal) (Independent Publisher)

### PR DESCRIPTION
**Pull request by troystaylor**
brave_GRqGT5LlJ7
brave_89awIl8yIG

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@troystaylor Thanks for your connector submission, we need to work with our internal teams to get AAD access to get the client id and secret for the connector, so please expect a delay in the certification of this connector. We will keep you posted with the updates. Thanks

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @troystaylor,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @troystaylor,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

File: independent-publisher-connectors/OneNote (Personal)/readme.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# OneNote (Consumer)
+Microsoft OneNote is your digital notebook. One cross-functional notebook for all your notetaking needs.
+
+## Publisher: Troy Taylor, Hitachi Solutions
+
+## Prerequisites
+Use of this connector requires access to Microsoft Graph, which will require an Azure subscription with Azure Active Directory. Follow these [instructions](https://docs.microsoft.com/en-us/graph/auth-register-app-v2) to configure your app registration.
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OneNote (Personal)/readme.md:7]: @natalie-pienkowska FYI - This connector needs Graph API access, we probably need to create a first-party app to request access from the Graph API team and that would take some time like the other IP connectors we are working on.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@troystaylor We need to work with Graph API team by creating a first app party internally to get client creds. So, please expect delays in your connector certification process as we need to deal with multiple internal teams. Thanks

**Comment by mamurshe** [comment]
@troystaylor while answering a question with the OneNote team I needed to find the difference between the standard OneNote Connector and this one. From the documentation: https://docs.microsoft.com/en-us/connectors/onenote/ it seems like all of these actions are already present in the standard connector. Can you please help me understand the difference between your connector and the standard one?

**Comment by troystaylor** [comment]
> @troystaylor while answering a question with the OneNote team I needed to find the difference between the standard OneNote Connector and this one. From the documentation: https://docs.microsoft.com/en-us/connectors/onenote/ it seems like all of these actions are already present in the standard connector. Can you please help me understand the difference between your connector and the standard one?

The OneNote (Business) connector cannot connect to notebooks created with a personal account. That’s the reason why this connector is badly needed. I have no idea why it has been overlooked for so long.

**Comment by Amjed-Ayoub** [comment]
Hello @troystaylor, I hope you are doing well. I just want to inform you that we are still working on getting internal authorization for your connector. Apologies for taking longer time than usual and if this is causing any inconvenience.
We will keep you posted.
Thank you so much for your patience and understanding.

**Comment by Amjed-Ayoub** [comment]
Hello @troystaylor,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by sriyen-msft** [comment]
@troystaylor Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**